parse.go

```diff
@@ -43,6 +43,7 @@ type JobStep struct {
 // StepWith is a (simplified) representation of a job step's `with:` object.
 type StepWith struct {
 	Script string `yaml:"script"`
+	Tag    string `yaml:"tag"`
```

Comment:
I don't think it will scale well to keep growing this struct with more inputs as more actions are supported. Instead, I think it makes more sense to change this into a `map[string]string`.

Reply:
Resolved in 112050b37ba081805696236f6a3f36c046395457